latest_cov_build.py

```diff
@@ -36,6 +36,11 @@ def get_github_release_url(repo_slug):
 taskId = index.findTask('gecko.v2.mozilla-central.' +
                         'latest.firefox.linux64-ccov-debug')['taskId']
 
+# Get revision of the build
+r = requests.get('https://queue.taskcluster.net/v1/' + 'task/{}'.format(taskId))
```

Comment:
No need to concatenate the strings, you can do `'https://queue.taskcluster.net/v1/task/{}'.format(taskId)` directly.